pkg/policy/groups/aws/aws.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package aws
+
+import (
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/aws/external"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/cilium/cilium/pkg/policy/api"
+)
+
+const (
+	awsLogLevel         = aws.LogOff // For debugging pourposes can be set to aws.LogDebugWithSigning
+	awsDefaultRegionKey = "AWS_DEFAULT_REGION"
+	awsDefaultRegion    = "eu-west-1"
```

Comment:
Why do we have a default region?

Reply:
Is a todo function that will happen in a follow-up PR.